src/service.c

```diff
@@ -387,14 +394,51 @@ static void onDispose(GObject *pObject)
     g_clear_object (&self->pPrivate->pActionGroup);
     g_clear_object (&self->pPrivate->pConnection);
 
+    if (m_pLibHandle)
+    {
+        dlclose(m_pLibHandle);
+        m_pLibHandle = NULL;
+    }
+
     G_OBJECT_CLASS(indicator_keyboard_service_parent_class)->dispose(pObject);
 }
 
 static void indicator_keyboard_service_init(IndicatorKeyboardService *self)
 {
+    gchar *sLib = "libayatana-keyboard-x11.so";
+    GFile *pFile = g_file_new_for_path("/usr/lib/maliit/plugins/liblomiri-keyboard-plugin.so");
```

Comment:
You sure, that this file is always in this location?

Should we make this more configurable for package maintainers, e.g. via a ``-DMALIIT_PLUGINS_DIR=...``?

I'll merge this PR anyway, but this hard-coded path smells like an upcoming problem.

Reply:
You can hold the merge until I get some more info on this...

Reply:
Damn, two seconds late!